packages/platforms/quanta/x86-64/x86-64-quanta-ix1-rangeley/onlp/builds/src/x86_64_quanta_ix1_rangeley/module/src/sysi.c

```diff
@@ -160,7 +160,7 @@ update_rpsu_fan_status(void){
     last_status = led_control.psu1_present;
     rv = onlp_gpio_get(QUANTA_IX1_PSU_GPIO_PSU1_PRSNT_N, &value);
     if(rv < 0) {
-        printf("GPIO %d read Error!\n", QUANTA_IX1_PSU_GPIO_PSU1_PRSNT_N);
+        AIM_LOG_ERROR("GPIO %d read Error!\n", QUANTA_IX1_PSU_GPIO_PSU1_PRSNT_N);
```

Comment:
@jonathantsai-qci  One small detail -- the AIM_LOG_* macros do not expect newlines. 
Can you remove the newlines in the messages? 
Then I think we're ready to go.